tests/integrated/test-beuler/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+bout_add_integrated_test(test_beuler SOURCES test_beuler.cxx)
```

Comment:
```suggestion
bout_add_integrated_test(test_beuler SOURCES test_beuler.cxx
  REQUIRES BOUT_HAS_PETSC)
```

Reply:
Does this need fixing in the v4.4.0-alpha port?